app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -223,10 +223,6 @@ class BloodPressureEntrySheetController @Inject constructor(
   }
```

Comment:
Please change the commit message to be more descriptive.
`Save BP when save is clicked in BloodPressureEntryScreen`